controllers/video.go

```diff
@@ -8,27 +8,38 @@ import (
 )
 
 type variantsResponse struct {
-	Name  string `json:"name"`
-	Index int    `json:"index"`
+	Name               string `json:"name"`
+	Index              int    `json:"index"`
+	VideoBitrate       int    `json:"VideoBitrae"`
+	IsVideoPassthrough bool   `json:"IsVideoPassthrough"`
```

Comment:
The frontend web client shouldn't need to know the specifics around the bitrate or what passthrough is.  If you're only using these values for sorting, can you change them to be `json:"-"` ?

Reply:
And don't export them `VideoBitrate` -> `videoBitrate`, etc.

Reply:
> The frontend web client shouldn't need to know the specifics around the bitrate

VideoJs does know this because ffmpeg writes it in the master hls playlist.

Reply:
> VideoJs does know this because ffmpeg writes it in the master hls playlist.

I'm talking specifically about this API response that was changed.  Not playback.